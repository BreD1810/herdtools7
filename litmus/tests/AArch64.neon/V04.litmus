AArch64 V04
(* Tests LD1 multiple structure *)
{
x=65535;
0:X0=x; 0:X1=y; 0:X2=z;
}
 P0                 ;
 LD1 {V0.8B},[X0]    ;
 ST1 {V0.B}[0],[X1] ;
 ST1 {V0.B}[1],[X2] ;
forall(y=255 /\ z=255)
