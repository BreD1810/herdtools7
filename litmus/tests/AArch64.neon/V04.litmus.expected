Test V04 Required
Histogram (1 states)
1000000:>y=255; z=255;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=255 /\ z=255) is validated
Hash=140b1b11a5de58a841f104420556db2e
Observation V04 Always 1000000 0
